packages/ui/components/plugin-grid/plugin-grid.tsx

```diff
@@ -55,25 +58,59 @@ function PluginTile({ icon, title, description, href }: PluginTileProps): JSX.El
           `,
         }}
       />
-      <div>
-        <img className="mb-2 h-[75px] w-[75px]" src={icon} alt={`${title} icon`} />
-        <h3>{title}</h3>
-        <p>{description}</p>
+      <div className="flex h-full flex-col justify-between">
+        <div>
+          <img className="mb-4 h-[75px] w-[75px]" src={icon} alt={`${title} icon`} />
+          <h3>{title}</h3>
+          <p className="line-clamp-3">{description}</p>
+        </div>
+        <div className="bg-pf-cyan-300 w-fit rounded px-2 py-1 text-sm text-white">{category}</div>
```

Comment:
![image](https://github.com/janus-idp/janus-idp.github.io/assets/7453394/0271a8ba-b4a1-4833-8779-814fea1b1982)


The tag on the plugin tile looks more like a button to me than a "tag" information. Also since "Frontend" and "Backend" strings look so similar when you glance over them it doesn't help me with quick orientation.

WDYT about changing this to a small React icon (for frontend), NodeJS icon (for backend) and some arbirtrary "action" or "automation" icon (for scaffolder actions) instead? All with tooltips.

Something like this? WDYT?
![image](https://github.com/janus-idp/janus-idp.github.io/assets/7453394/f60e08c6-6ce4-4f95-95b1-42c05cc10d40)

Reply:
I like this idea. Thoughts @serenamarie125?